cli/tests/test_git_colocated.rs

```diff
@@ -130,6 +130,8 @@ fn test_git_colocated_unborn_branch() {
     let (stdout, stderr) = test_env.jj_cmd_ok(&workspace_root, &["checkout", "root()"]);
```

Comment:
nit: Update this test case to use `jj new` instead? it's not meant to test `jj checkout`.

The same is true for part of the `test_checkout_conflicting_branches()` and `test_checkout_conflicting_change_ids()` above. I think we'd ideally move those to `test_conflict.rs` before this commit, but I won't insist if you don't have time for it.

Reply:
Oh, sorry to have wasted your time, moving them to `test_new_command.rs` is what I meant!